core/src/consensus.rs

```diff
@@ -127,22 +127,42 @@ pub const MAX_BLOCK_WEIGHT: usize = 40_000;
 /// Fork every 6 months.
 pub const HARD_FORK_INTERVAL: u64 = YEAR_HEIGHT / 2;
 
+/// Floonet first hard fork height, set to happen around 2019-06-20
+pub const FLOONET_FIRST_HARD_FORK: u64 = 184_320;
```

Comment:
For the record this will happen on 2019-06-19 22:56:46. Can we move it a bit around June 20?

Reply:
Are you assuming an 60s block time average? 
Not sure we have the graph rate consistency for that.
I often see average block times well over 60s.
I'd say this is more likely to hit on June 20 than on June 19...

Reply:
Yeah, assuming a constant 60s per block. Starting at block height 166_469 on 2019-06-07 13:25:46 UTC.